components/atom/popover/src/index.scss

```diff
@@ -0,0 +1,71 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$fz-atom-popover: $fz-m !default;
+$lh-atom-popover: $lh-m !default;
+$c-atom-popover: $c-black !default;
+$bdc-atom-popover: $c-gray-light-3 !default;
+$bgc-atom-popover: $c-white !default;
+$bdrs-atom-popover: $bdrs-m !default;
+$bdw-atom-popover: 1px !default;
+$pt-atom-popover: $p-m !default;
+$pb-atom-popover: $p-m !default;
+$pl-atom-popover: $p-m !default;
+$pr-atom-popover: $p-m !default;
+$m-atom-popover: 0 !default;
```

Comment:
Without seeing the Demo: is it possible that margin and paddings are inverted?
I mean Padding (space within the popover) should be 0, and the Margins (space to the parent element) 8px?

Reply:
@turolopezsanabria you can view the demo [here](https://sui-components-pr-891-qlpvpzyaur.now.sh/workbench/atom/popover/demo)